packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -14,11 +14,1064 @@ The Okta Administrator Roles API provides operations to manage administrative ro
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+```
+
+##### Response example
+
+```json
+{
+  "resources": [
+    {
+      "id": "ire106sQKoHoXXsAe0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    },
+    {
+      "id": "ire106riDrTYl4qA70g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00gu67DU2qNCjNZYO0g3/users"
+        }
+      }
+    },
+    {
+      "id": "irezvo4AwE2ngpMw40g3",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/users"
+        }
+      }
+    },
+    {
+      "id": "irezvn1ZZxLSIBM2J0g3",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "next": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources?after=irezvn1ZZxLSIBM2J0g3"
+    },
+    "resource-set": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    }
+  }
+}
+```
+
+#### Delete a resource
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}/resources/${resourceId}" />
+
+Remove a resource from a resource set
+
+##### Request parameters
+
+| Parameter      | Description                             | Param Type   | DataType     | Required |
+| :------------- | :-------------------------------------- | :----------- | :----------- | :------- |
+| resourceSetId  | id of the resource set                  | URL          | String       | TRUE     |
+| resourceId     | id of the resource within resource set  | URL          | String       | TRUE     |
+
+`resourceId` is the id obtained when [listing resources within resource set](#list-resources). For example, if the resource object was:
+```json
+    {
+      "id": "ire106sQKoHoXXsAe0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    }
+```
+`ire106sQKoHoXXsAe0g4` could be used as `resourceId` to remove the groups from list of resources in the set.
+
+##### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources/ire106sQKoHoXXsAe0g4"
+```
+
+##### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+## Custom role assignment operations
+<ApiLifecycle access="beta" />
+These operations allow the assignment and unassignment of custom roles. This is done by creating a binding.
+A binding is a map withe the key being a role and value being a list of users/groups that get the role.
```

Comment:
this is true in the JSON document model for thinking about policy, but is not true as exposed by this API and the DB - i.e. each binding is a unique entity relationship of BindingID -> Principal:Role:ResourceSet.

Reply:
did some rewording here